manifest.json

```diff
@@ -41,5 +41,13 @@
     "projectOptions": ["app", "addon"],
     "nodeVersion": "6.0.0",
     "commands": ["jscodeshift -t https://raw.githubusercontent.com/simplabs/qunit-dom-codemod/master/qunit-dom-codemod.js ./tests"]
+  },
+  "ember-angle-brackets-codemod": {
+    "versions": {
+      "ember-cli": "3.10.0-beta.1"
+    },
+    "projectOptions": ["app", "addon"],
+    "nodeVersion": "6.0.0",
```

Comment:
Judging by https://github.com/ember-codemods/ember-angle-brackets-codemod/commit/b45c923e7167ee6b6c83fa09ac7fa3211988fc21#diff-354f30a63fb0907d4ad57269548329e3R3, I think this should be 8.0.0